sdks/python/apache_beam/transforms/cy_combiners.py

```diff
@@ -405,3 +408,22 @@ def increment_bucket(self, bucket_index):
 
 class DistributionCounterFn(AccumulatorCombineFn):
   _accumulator_type = DistributionAccumulator
+
+
+class DistributionMetadataAccumulator(object):
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Can you please add a class docstring describing how this is to be used?--it is not obvious from reading this file.  This looks like just a generic list accumulator to me.

Reply:
Actually, it's just a list. will it be much clearer using another class name such as Int64ListAccumulator?

Reply:
Can you add a docstring?